README.md

```diff
@@ -17,3 +17,28 @@ Usage:
     make DEST=<destination path>
 
 If DEST is not set, package will stay in current directory
+
+## Kernel Configuration Changes
+
+The Debian kernel used with SONiC includes almost all available hardware that can be found on a system using Linux. This increases considerably the time needed to build the kernel Debian image. Since there are many drivers, protocols or filesystems which will never be used on a switch, there is a simple mechanism to remove kernel options which does not require creating a kernel configuration patch. The options which need to be excluded from the kernel are simply listed in a flat text file:
```

Comment:
how much saving here in terms of compile speed and the linux-image deb size? can you provide data?

Reply:
How can I attach a docx, pdf or other document here, in the comment section?
Each time I tried, I got "Something went really wrong, and we can't process that file."
I have a spreadsheet that describes which kernel modules were removed in this patch.

Reply:
Please take look at this document which describes which modules would be removed:
https://gist.github.com/olivier-singla/e1aaafba63756f9fcdac13054ee5a8de

Reply:
can you submit this doc as part of the pr. We can have a doc folder for this doc.